app/View/Elements/tutorial_content.ctp

```diff
@@ -16,7 +16,7 @@
     <div id="navigation" class="clearfix">
       <div id="prev-navigation">
 			<?php echo $this->Html->link(
-				__('Previous'),
+				__(''),
```

Comment:
Ah, yes. That makes sense. I guess when I was looking at it I was thinking it was part of the function. I'll remove that.